.ci.sh

```diff
@@ -30,14 +30,7 @@ echo *** ANALYZE CODE ***
 pip install pylint
 bash -c 'PYTHONPATH="$PWD/scripts:$PWD" pylint -j3 --py3k --output-format=parseable --errors-only $(git ls-tree --name-only -r HEAD | sort | grep -E "(spinalcordtoolbox|scripts|testing).*\.py" | xargs); exit $(((($?&3))!=0))'
 
-#
-# echo *** BUILD DOCUMENTATION ***
-# pip install sphinx sphinxcontrib.programoutput sphinx_rtd_theme
-# cd documentation/sphinx
-# make html
-# cd -
-
-# python create_package.py -s ${TRAVIS_OS_NAME}  # test package creation
-# cd ../spinalcordtoolbox_v*
-# yes | ./install_sct  # test installation of package
-
+if [[ "$BRANCH" == "master" ]]; then
```

Comment:
Maybe `$TRAVIS_BRANCH` to be more explicit? :shrug:

Reply:
`TRAVIS_BRANCH` is actually something else. See (https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables). Relevant section:
![image](https://user-images.githubusercontent.com/7751484/94429160-e591f180-015f-11eb-810d-0dcbb67357c5.png)

So based on this logic, when testing PRs, `TRAVIS_BRANCH` == target branch == `master`. This is the reason for the little gymnastics in `.ci.sh` to figure out if we are in a PR or not and use the proper env var.